src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.ArrayList;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ComplexCommand implements DriverCommand{
+    ArrayList<DriverCommand> list;
+    
+    public ComplexCommand(ArrayList<DriverCommand> newList) {
+    	list = newList;
+    }
+    
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.